tools/tpm2_createpolicy.c

```diff
@@ -322,6 +359,21 @@ static bool init(int argc, char *argv[], createpolicypcr_ctx *ctx) {
                 return false;
             }
             break;
+        case 'C':
+            snprintf(context_key_file, sizeof(context_key_file), "%s", optarg);
```

Comment:
We don't need to be copying things.. see bug: https://github.com/01org/tpm2.0-tools/issues/260. Just assign it and use it.

char *context_key_file_path = optarg;